packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -280,6 +318,30 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
         expect(o.testState.cloudProjectRequestAccessWasCalled).to.eql(true)
       })
     })
+
+    it('updates the button text when the request access button is clicked', () => {
+      cy.remoteGraphQLIntercept(async (obj, testState) => {
+        if (obj.operationName === 'Runs_currentProject_cloudProject_batched') {
+          for (const proj of obj!.result!.data!.cloudProjectsBySlugs) {
+            proj.__typename = 'CloudProjectUnauthorized'
+            proj.message = 'Cloud Project Unauthorized'
+            proj.hasRequestedAccess = false
+            testState.project = proj
+          }
+        } else if (obj.operationName === 'RunsErrorRenderer_RequestAccess_cloudProjectRequestAccess') {
+          obj!.result!.data!.cloudProjectRequestAccess = {
+            ...testState.project,
+            hasRequestedAccess: true,
+          }
+        }
+
+        return obj.result
+      })
+
+      cy.visitApp('/runs')
+      cy.findByText(defaultMessages.runs.errors.unauthorized.button).click()
+      cy.findByText(defaultMessages.runs.errors.unauthorizedRequested.button).should('exist')
```

Comment:
Just curious, this the context of this test, but is the following describe block of `'Runs - Unauthorized Project Requested'` testing something different compared to the current describe block of `Runs - Unauthorized Project`? 
 Wondering if we just need to enhance one of these?

Reply:
AFAIK in `'Runs - Unauthorized Project Requested'`, we're validating the correct UI appears for users who have previously requested access, whereas in `Runs - Unauthorized Project`, we're testing the request access flow itself